### ISSUE TITLE
Organise endpoints to fastapi alternative of `cogs`

### ISSUE THREAD

**Issue by Dhravya** [Dhravya/dhravyaAPI#39]
Not sure if its possible, but currently the code is really really long and unorganised, it is possible to have everything in different files?

**Comment by itsnotrin** [comment]
Yeah, it's something called routers, we could also use that to organize things into topics like /fun/endpoint here etc
